index.html

```diff
@@ -0,0 +1,161 @@
+                <img src="imgs/facebook.svg" width="20px" alt="facebook icon">
+                <img src="imgs/twitter.svg" width="20px" alt="twitter icon">
+                <span  class="text-muted">English</span>
+                <span  class="text-muted">My Page</span>
+                <span  class="text-muted me-5">Logout</span>
+            </div>
+        <section class="header container-fluid bg-white ">
+            <navbar class="d-flex justify-content-between align-items-baseline mt-sm-3 p-0 m-0 gap-5">
+                <div class="d-none d-md-block d-lg-block fs-5 ms-5 mob-logo"> 
+                      <a href="#">
+                      <img width="120px" src="./imgs/logo.jpg" alt="Techno logo" class="bg-light" />
+                      </a>
+                </div>
+                <div class="hamburger d-sm-flex flex-column d-md-none d-lg-none p-3">
+                    <span class="d-block bar"></span>
+                    <span class="d-block bar"></span>
+                    <span class="d-block bar"></span>
+                </div>
+                <div class="d-none d-md-block me-5">
+                    <ul class="d-flex  justify-content-end align-items-baseline p-1 gap-5 justify-content-between me-5">
+                        <li class="list-inline "><a href="about.html" class="text-decoration-none nav-link ">About</a></li>
+                        <li class="list-inline "><a href="#" class="text-decoration-none nav-link">Program</a></li>
+                        <li class="list-inline "><a href="#" class="text-decoration-none nav-link">Join</a></li>
+                        <li class="list-inline "><a href="#" class="text-decoration-none nav-link">Sponser</a></li>
+                        <li class="list-inline "><a href="#" class="text-decoration-none nav-link">News</a></li>
+                        <a href="#" class="p-2 list-inline text-decoration-none border border-5 border-danger"><b class="bg-black text-muted align-content-center p-1">Buy Ticket</b></a>
+                    </ul>
+                </div>
+            </navbar>
+        </section>
+            
+            <ul class="menu">
+                <li class="nav-item"><a href="about.html" class="nav-link ">About</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponser</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+            </ul>
+    
+
+        <section class="headline d-flex flex-column gap-1 justify-content-between  align-items-baseline ">
+            <span class="size font-color text-xl font-medium text-md-3xl fw-semibold ">"Hello! Tech Lover"</span>
+            <div ><h1 class="h-grd fw-bolder ">THE TECH-No GLOBAL<br> ANNUAL EVENT 2023</h1></div>
```

Comment:
- [ ] [OPTIONAL] I see that you seem to be having difficulty in accessing the correct font family for this title. Here is it: `@import url("http://fonts.cdnfonts.com/css/cocogoose");` You can simply import in your CSS file. This will really make your design more appealing and aesthetic. :+1: